AnalyzerTools/src/MCCorrection.C

```diff
@@ -174,6 +174,54 @@ void MCCorrection::SetMCSample(TString s){
 void MCCorrection::SetDataYear(int i){
   DataYear = i;
 }
+void MCCorrection::SetIsFastSim(bool b){
+  IsFastSim = b;
+}
+
+double MCCorrection::MuonReco_SF(TString key, double eta, double p, int sys){
```

Comment:
in header it is labelled as pt, is it pt or p?

Reply:
It will be different what is provided by MuonPOG, but for "HighPtMuonRecoSF" it is p. But I will change the header for now